### DISCUSSION TITLE
error on deploy nestjs app

### DISCUSSION THREAD

**Discussion by alexucgit** [vercel/vercel#9054] [Help]
Hi!
When build my app I've this error. 
```
+0ms
2022-12-09T13:07:33.501Z	3b737f60-e047-422a-9d6b-29b6a6653ac9	ERROR	No exports found in module "../src/main.js".
2022-12-09T13:07:33.501Z	3b737f60-e047-422a-9d6b-29b6a6653ac9	ERROR	Did you forget to export a function or a server?
RequestId: 3b737f60-e047-422a-9d6b-29b6a6653ac9 Error: Runtime exited with error: exit status 1
Runtime.ExitError
```

and when I run app I've this https://agmbe.vercel.app

**Comment by amyegan** [comment]
Thanks for reaching out @alexucgit! I need a little more information. Do you see additional error details in the [function logs](https://vercel.com/guides/where-are-vercel-logs)? 

If you are able to share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that is usually the best way for us to see exactly what's going wrong and identify possible solutions.

**Reply by alexucgit** [reply]
This is my project 
https://github.com/alexucgit/agmbe

**Reply by alexucgit** [reply]
my Vercel.json is

{ "version": 2, "builds": [ { "src": "src/main.ts", "use": "@vercel/node" } ], "routes": [ { "src": "/(.*)", "dest": "src/main.ts", "methods": ["GET", "POST", "PUT", "PATCH", "DELETE"] } ] }

**Reply by alexucgit** [reply]
> Thanks for reaching out @alexucgit! I need a little more information. Do you see additional error details in the [function logs](https://vercel.com/guides/where-are-vercel-logs)?
> 
> If you are able to share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that is usually the best way for us to see exactly what's going wrong and identify possible solutions.

GET] /favicon.ico
16:51:24:04
[Nest] 8  - 
12/10/2022, 3:51:25 PM 
    LOG
 
[NestFactory] 
Starting Nest application...
[Nest] 8  - 
12/10/2022, 3:51:25 PM 
    LOG
 
[InstanceLoader] 
TypeOrmModule dependencies initialized
 +122ms
[Nest] 8  - 
12/10/2022, 3:51:25 PM 
    LOG
 
[InstanceLoader] 
PassportModule dependencies initialized
 +0ms
[Nest] 8  - 
12/10/2022, 3:51:25 PM 
    LOG
 
[InstanceLoader] 
JwtModule dependencies initialized
 +1ms
[Nest] 8  - 
12/10/2022, 3:51:25 PM 
    LOG
 
[InstanceLoader] 
AppModule dependencies initialized
 +0ms
2022-12-10T15:51:30.035Z	f2b0b728-b8b5-4630-8a58-104a10153abf	ERROR	No exports found in module "../src/main.js".
2022-12-10T15:51:30.035Z	f2b0b728-b8b5-4630-8a58-104a10153abf	ERROR	Did you forget to export a function or a server?
RequestId: f2b0b728-b8b5-4630-8a58-104a10153abf Error: Runtime exited with error: exit status 1
Runtime.ExitError
